config/redirects.js

```diff
@@ -1683,11 +1683,19 @@ module.exports = [
     to: '/integrations/sso/zoom'
   },
   {
-    from: `/guides/login/centralized-vs-embedded`,
-    to: `/guides/login/universal-vs-embedded`
+    from: '/guides/login/centralized-vs-embedded',
+    to: '/guides/login/universal-vs-embedded'
   },
   {
-    from: `/guides/login/migration-embedded-centralized`,
-    to: `/guides/login/migration-embedded-universal`
+    from: '/guides/login/migration-embedded-centralized',
+    to: '/guides/login/migration-embedded-universal'
+  },
+  {
+    from: '/api/management/v2/user-search',
+    to: '/users/search'
```

Comment:
Since these two do not have the `v2` in the URL, they will redirect to the default one, which is v3. Is this intentional?